cookiecutter.json

```diff
@@ -1,4 +1,15 @@
 {
-  "plugin_name": "example_plugin",
-  "plugin_class_name": "ExamplePlugin"
+  "readable_plugin_name": "Example Plugin",
+  "description": "",
+  "repository_url": "",
+  "author": "Learning Equality",
+  "email": "info@learningequality.org",
+  "license": ["MIT", "GNU GPL v3.0", "Apache Software License 2.0", "The Unlicense"],
```

Comment:
Comment from @jamalex that we probably want to remove GPL from here as it makes reusing others' plugins problematic.

Can also collapse to a hard default of MIT that people have to modify if they want to change the license.